app/views/root/index.html.erb

```diff
@@ -48,9 +48,6 @@
 
       <section class="services-block" aria-labelledby="services-and-information-label" id="services-and-information">
         <div class="inner-block floated-children">
-          <div class="two-column-heading">
-            <h2 id="services-and-information-label">Services and information</h2>
```

Comment:
I think making the `h2` visually hidden makes a lot of sense, thanks.  I'll gather feedback on what a good improvement to the working might be.